application/CMakeLists.txt

```diff
@@ -118,7 +122,7 @@ set_target_properties(gpms_ui PROPERTIES
 
 # Include directories
 target_include_directories(${PROJECT_NAME} PRIVATE
-    ${OpenCV_INCLUDE_DIRS}
+    ${_INCLUDE_DIRS}
```

Comment:
What is this change doing? Will there be any change to set-up in the IDE we'll need to address?

Reply:
Hmm, I'm not exactly sure. I pulled from main, idk if that was one of the things that Francisco changed, idk if it works like that. I tried to see what changed but I did the command wrong. It still worked though. Good thing you saw this but I am really not sure.

Reply:
Or it could've been when I changed the whole CMake to be from a specific directory when I was having issues in class on Friday with it building the wrong project.

Reply:
Okay I just checked, neither of those things do anything. They are supposed to be variables. I must have deleted them but left it there.

Reply:
Well, I thought they might've been enviorment variables that are set-up through QT